src/pipecat/frames/frames.py

```diff
@@ -393,6 +404,25 @@ class StopInterruptionFrame(SystemFrame):
     pass
 
 
+@dataclass
+class UserStartedSpeakingFrame(SystemFrame):
```

Comment:
This is now a SystemFrame. Not actual effect from before because it was the first frame pushed after a `StartInterruptionFrame` so there were no queues.